.bazelignore

```diff
@@ -1,2 +1,7 @@
 e2e/
-examples/node_modules/
+examples/assert_lockfile_up_to_date/node_modules/
```

Comment:
nit: could add a test that this file and pnpm-workspace.yaml agree on the node_modules folder locations

Reply:
agreed; it is annoying that it is can easily fall out of date; users should do the same

Reply:
will make an issue to do as a follow-up

Reply:
https://github.com/aspect-build/rules_js/issues/196